frontend-shared/styles/mixins/utils.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Fast transition effect with fixed speed and ease.
+ *
+ * @param {string[]} $properties - CSS properties, such as background-color, to apply the
+ *   transition too.
+ */
+@mixin transition--fast($properties...) {
```

Comment:
I decided to go with a rapid transition on the box-shadow so this mixin helps manage these transitions because many elements also need to apply it on background-color and color.